### ISSUE TITLE
[SF][A11Y][Breadcrumbs]: "current page" Announcement is Missing 

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5198]
**Describe the bug**

In screen reader, "current page" announcement is missing for the current page in the trail when focus on current location.

**To Reproduce**
1. Go to https://qacand.hcm.ondemand.com/sf/OMP/Mentoring/MentorPreference?from_module=omp&p=mentee&programId=77355&appType=mentoringProgram&_s.crb=4aq%252b4U11%252frke1YIgTOier45VWCJZwC4cBVnz2TD7gpE%253d#/mentor-program/mentee/77355 
2. Using JAWS navigate through the breadcrumb trail 
3. See error: "Program details" is the current page but the "current page" announcement is missing when focus is on program details in the breadcrumb trail
![image](https://user-images.githubusercontent.com/78520316/167752460-2ef54559-0377-4ebe-ac68-e67393532be6.png)

**Expected behavior**

When focus on the current page list item in the trail, "current page" announcement is necessary. 
Please refer to the open UI5 example in which all announcements are as expected
![image](https://user-images.githubusercontent.com/78520316/167753542-3652fdb3-41eb-47d1-959d-76c9888d1c67.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: "1.3.1",
`@ui5/webcomponents-react` version: "0.23.2"

Operating System: Windows 10 
Browser: Chrome 100

**Additional context**

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by niyap** [comment]
Hello @sfsf-xweb-sh,

May I ask you to provide credentials via the internal channels (slack).

Thank you in advance!

Kind Regards,
Niya

**Comment by sfsf-xweb-sh** [comment]
Hi @niyap  I can't found you in slack,

**Comment by hristop** [comment]
Hi colleagues ,

Can you please look into this? The issue is reproducible on the master as well. 

Best Regards,
Hristo
